pagination/src/main/java/com/cerner/beadledom/pagination/parameters/exceptions/InvalidParameterException.java

```diff
@@ -0,0 +1,40 @@
+package com.cerner.beadledom.pagination.parameters.exceptions;
+
+import com.cerner.beadledom.json.common.model.ErrorDetail;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Exception used to communicate an invalid parameter.
+ *
+ * <p>Contains metadata used to build {@link Error} and {@link ErrorDetail} for the resulting
+ * JSON response.
+ *
+ * @author Brian van de Boogaard
+ * @since 3.1
+ */
+public final class InvalidParameterException extends RuntimeException {
```

Comment:
Since we may want to eventually move the `AbstractParameter` into another module for more reusability, what do you think about making `AbstractParameter` package-private and moving this exception up to the same package as `AbstractParameter` and making this exception class package-private as well?

That could give us more freedom to move things around in the future.

Reply:
Sounds, good. Will do that now.